README.md

```diff
@@ -4,12 +4,12 @@
   - [Terminology](#terminology)
   - [Teams & Roles](#teams--roles)
     - [Steering Council](#steering-council)
+    - [Sub-Teams](#sub-teams)
+      - [Dynamic Charter](#dynamic-charter)
+      - [Static Charter](#static-charter)
     - [Project Teams](#project-teams)
     - [External Contributors](#external-contributors)
     - [Emeritus Steering](#emeritus-steering)
-  - [Sub-Teams](#sub-teams)
-    - [Dynamic Charter](#dynamic-charter)
-    - [Static Charter](#static-charter)
   - [Community, Federated & Archive Maintenance](#community-federated--archive-maintenance)
     - [Community Projects](#community-projects)
     - [Federated Projects](#federated-projects)
```

Comment:
I think we should get rid of this ToC while we're at it, since it's quite the distraction (have gotten that feedback from a number of people) and GitHub already renders a ToC [since last years](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/).

Reply:
Ah very good.  I can drop the manual TOC. 

I thought it was clearer to have the sub-teams under the Teams & Roles section.  It could have stayed where it was, but that struck me as sub-optimal.

Reply:
And dropped

Reply:
Thank you!